.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
+  - name: "💬 Chat"
+    url: https://matrix.to/#/#jenkins-release:libera.chat
+    about: Please ask and answer questions here.
```

Comment:
maybe one of 
```suggestion
    about: Please ask and answer questions on Matrix (or one of the bridges e.g. libera IRC or gitter) (#jenkins-release).
```

```suggestion
    about: Please ask and answer questions on Matrix (#jenkins-release).
```

Reply:
I'd be in favor of the second one.

Reply:
The first mentions IRC though, I'd go with it as it's more descriptive.